src/api/nodeRpcCall.js

```diff
@@ -1334,6 +1334,48 @@ const applyForCongress = async (walletAddress, callback) => {
   });
 }
 
+const createLlmSpendingMotion = async ({ walletAddress, transferToAddress, transferAmount, spending }, callback) => {
```

Comment:
```suggestion
const congressSendLlm = async ({ walletAddress, transferToAddress, transferAmount, spending }, callback) => {
```

There are 2 other PRs already that use `congressDoAction` naming for creating congress motions: https://github.com/liberland/liberland_frontend/pull/117/files#diff-23f1d917744ae222dc753c21f0aea86fd32bbd65ef4e552d1650bba52573b51eR1608
As those PRs aren't merged yet I'm happy to choose some other pattern, but lets figure out something consistent.